src/svgdevicecontext.cpp

```diff
@@ -253,7 +259,7 @@ void SvgDeviceContext::StartPage()
     m_currentNode.append_attribute("class") = "page-margin";
     m_currentNode.append_attribute("transform")
         = StringFormat("translate(%d, %d)", (int)((double)m_originX), (int)((double)m_originY)).c_str();
-    m_currentNode.append_attribute("style") = "stroke: #000; stroke-opacity: 1.0; fill: #000; fill-opacity: 1.0";
+    m_currentNode.append_attribute("style") = "stroke: #000; fill: #000;";
```

Comment:
`fill-opacity`, `stroke-width` and `stroke-opacity` have all an initial value of 1 and they get inherited. See [here](https://www.w3.org/TR/SVG/painting.html).